www/public/index.html

```diff
@@ -7,6 +7,9 @@
         <meta name="theme-color" content="#000000" />
         <meta name="description" content="Web site created using create-react-app" />
         <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+        <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
```

Comment:
Is this the font we are standardising on or is this just temporary?

Reply:
For now I wanna standardize to use Roboto font family. That's what I used for the Figma mock ups. That being said I'm not that tied to it, if we find another font to use I'm fine with that. Just don't really wanna focus too much for now I suppose